tests/test_other.py

```diff
@@ -8229,3 +8229,9 @@ def test_autotools_shared_check(self):
       except OSError:
         # Ignore missing python aliases.
         pass
+        
+  def test_ioctl_window_size(self):
+      run_process([PYTHON, EMCC, path_from_root('tests', 'ioctl', 'ioctl_window_size.c')])
```

Comment:
these 4 lines can be replaced with
```
    self.do_other_test(os.path.join('other', 'ioctl'))
```
, but you'll need to rename the `c` file to `test.cpp` and the output file to `test.out`.

Reply:
Also, would be better to do `self.do_other_test(os.path.join('other', 'ioctl', 'window_size'))` (add a subdir `window_size`).

Reply:
Updated it. My local copy didn't have `self.do_other_test` so I pulled it from upstream -- you should be able to remove any changes this made while merging.